hive/src/main/java/com/netease/arctic/hive/op/HiveOperationTransaction.java

```diff
@@ -173,182 +158,4 @@ public void commit() {
       }
     }
   }
-
-  private class TransactionalTable implements Table {
```

Comment:
Why remove this `TransactionalTable` implementation? I think we need overwride transactional table `newOverwrite` and `newReplacePartitions` method?

Reply:
First, I find no use the TransactionalTable anywhere before.
I think use the transactionalTable in HiveOperationTransaction.wrapped is enough?